### PR TITLE
Zoma/simple prompt

### PR DESCRIPTION
**Pull request by coldplaying42**
Fixes #

## Proposed Changes

  - 
  -
  -

### CODE REVIEW

File: experimental/generation/generator/packages/library/templates/generator.lg [hunk 1]
```diff
@@ -424,4 +424,52 @@ ${substitutions(`library-operations.${locale}.data`,
 - ${designerName(`${property} - Missing()`)}
 
 # designerIntent(intent)
-- ${designerName(`On${intent}`)}
\ No newline at end of file
+- ${designerName(`On${intent}`)}
+
+# missingNumber(property)
```
Comment (chrimc62) [inline experimental/generation/generator/packages/library/templates/generator.lg:429]: This should always include autoHelp().

File: experimental/generation/generator/packages/library/templates/generator.lg [hunk 2]
```diff
@@ -424,4 +424,52 @@ ${substitutions(`library-operations.${locale}.data`,
...
+
+# missingStringArray(property)
+- ```\`\`\`
+\${getAutoHelpText()}
+What values do you want to add for ${getPropertyNameText(property)}?
+\`\`\`
+```
+
+# getPropertyNameText(property)
+- ${property}
```
Comment (chrimc62) [inline experimental/generation/generator/packages/library/templates/generator.lg:475]: >property [](start = 4, length = 8)

This should actually be ${phrase(property)}

**Comment by chrimc62** [comment]
Good start.  You should also bring in the help text from library-Help.en-us.lg to the per-property generated .lg.  Looks like there are also some cases missing there for the new format stuff.

**Comment by mrivera-ms** [comment]
Hello @coldplaying42 , this PR was closed accidentally during the process of renaming the branch 'master' to 'main'. Please open a new PR that references the new 'main' branch. I apologize for the inconvenience.

**Pull request closed without merging**